### ISSUE TITLE
wazuh-modulesd:logcollector do not appear when using SendMSGtoSCK method and Read_Localfile method

### ISSUE THREAD

**Issue by damarisg** [wazuh/wazuh#8927]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 5.0  | wazuh-modulesd:logcollector | Manager/Agent | Packages | CentOS 8|

While executing logcollector test, we've detected a problem with the prefix of the log. The test affected are:
- test_location_custom_sockets/test_location_custom_sockets.py
- test_configuration/test_basic_configuration_only_future_events.py 
- test_configuration/test_basic_configuration_reconnect_time.py  (Windows)
______________________________________________________________________________________________________________________________________

### Details:

Branch with changes about unification:  dev-agent-process-unified 
Packages generated from this branch.

Repository with test executed on [wazuh-qa](https://github.com/wazuh/wazuh-qa) from 5.0-dev
______________________________________________________________________________________________________________________________________
### Description

**To test_location_custom_sockets/test_location_custom_sockets.py**

On Wazuh 4.3 version execution, the following results can be seen:
![imagen](https://user-images.githubusercontent.com/3978434/121192308-33a8da00-c843-11eb-9f5f-751e20eb0929.png)

On Wazuh 5.0 version execution, the following not expected results were seen:
![imagen](https://user-images.githubusercontent.com/3978434/121192594-7b2f6600-c843-11eb-9446-a5c616d1da6a.png)

________________________________________________________________________________
**To  test_configuration/test_basic_configuration_only_future_events.py**

On Wazuh 4.3 version execution, the following results can be seen:
![imagen](https://user-images.githubusercontent.com/3978434/121237598-9b741a80-c86d-11eb-8d39-acb74354e626.png)

On Wazuh 5.0 version execution, the following not expected results were seen:
![imagen](https://user-images.githubusercontent.com/3978434/121237664-afb81780-c86d-11eb-9d23-cc64f39b9d78.png)

________________________________________________________________________________
**To test_configuration/test_basic_configuration_reconnect_time.py**
On Windows also should be "wazuh-modulesd:logcollector" but we found "wazuh-agent". 
We see that it is the same method of test_basic_configuration_only_future_events.py.

![imagen](https://user-images.githubusercontent.com/3978434/121434922-6ee6fe00-c954-11eb-91af-64e72f980220.png)

On Wazuh 5.0 version execution, the following not expected results were seen:
![imagen](https://user-images.githubusercontent.com/3978434/121434695-13b50b80-c954-11eb-8d54-4deead359741.png)

Taking into account the changes being done as part of the process unification work, the tests should passed considering the new logging tag: "wazuh-modulesd:logcollector" which, indeed, is not happening.

**Comment by tinachoB** [comment]
After a deeper looking into this, we've realized that this is not an error in the code but a need to change the specific tests mentioned.

The function `SendMSGtoSCK`does not belong to logcollector module at all. Instead, this function is a helper one which takes the current **process name** to be logged (as showed in the image in the bug description), in this case **wazuh-modulesd** (in previous versions, the process name was logcollector). We can't change this function because, as I said before, it's a helper one and we don't want to break any other module's behavior.

To sum up, the tests should be updated to check for wazuh-modulesd instead of logcollector.
